packages/vuetify/src/components/VSlideGroup/VSlideGroup.tsx

```diff
@@ -108,6 +109,50 @@ export const VSlideGroup = genericComponent<VSlideGroupSlots>()({
       return group.items.value.findIndex(item => item.id === group.selected.value[group.selected.value.length - 1])
     })
 
+    const scrollToPosition = (newPosition: number) => {
+      if (!IN_BROWSER || !containerRef.value) {
+        return
+      }
+
+      const offsetSize = getOffsetSize(containerRef.value)
+      const scrollPosition = getScrollPosition(containerRef.value)
+      const scrollSize = getScrollSize(containerRef.value)
+
+      if (scrollSize <= offsetSize) {
+        return
+      }
+
+      // Prevent scrolling by only a couple of pixels, which doesn't look smooth
+      if (Math.abs(newPosition - scrollPosition) < 16) {
+        return
+      }
+
+      if (isHorizontal.value) {
+        animateHorizontalScroll({
+          container: containerRef.value!,
+          left: newPosition,
+          rtl: isRtl.value,
+        })
+      } else {
+        animateVerticalScroll({
+          container: containerRef.value!,
+          top: newPosition,
+        })
+      }
```

Comment:
Why not https://developer.mozilla.org/en-US/docs/Web/API/Element/scrollTo?

Reply:
scrollTo is not well controlled and seems this bugs is still actual:

https://bugs.webkit.org/show_bug.cgi?id=238497
https://bugs.webkit.org/show_bug.cgi?id=220908


I recorded a comparison of how a native scrollTo with smooth behavior works versus a js implementation. scrollTo looks ugly by using left/right arrow to navigate.
https://www.loom.com/share/f75e7400bbe449068cef6f2b88d2a942

Reply:
238497 - only applies to `scroll-behavior: smooth` css, not `behavior: 'smooth'`?
220908 - unrelated, we already have a workaround for that in relevant components

> scrollTo looks ugly by using left/right arrow to navigate
Doesn't seem too bad, and it's only when you're holding the button down

Reply:
> Doesn't seem too bad, and it's only when you're holding the button down

Well, not much code but better + supports different transitions :)

Reply:
What is the conclusion of this conversation? Should we use scrollTo?